index.bs

```diff
@@ -2302,6 +2346,98 @@ To <dfn>parse trigger specs</dfn> given a [=map=] |map|, a [=moment=]
 Issue: Invoke [=parse summary buckets=] and [=parse summary window operator=]
 from this algorithm.
 
+To <dfn>parse max number of event states</dfn> from a [=map=] |map|,
+a [=source type=] |sourceType|,
+a possibly null 32-bit unsigned integer |attributionScopeLimit|,
+and a [=randomized response output configuration=] |config|:
+1. If |attributionScopeLimit| is null:
+    1. If |map|["`max_event_states`"] [=map/exists=], return an error.
```

Comment:
Please add new fields to https://wicg.github.io/attribution-reporting-api/#source-registration-json-key.

Reply:
Nice catch. Done.